scripts/interfaces.ts

```diff
@@ -0,0 +1,167 @@
+// Native
+import { readFile, writeFile } from "fs/promises";
+import { mkdirSync } from "fs";
+import { basename, parse, dirname } from "path";
+
+// Vendor
+import glob from "glob";
+
+// Utilities
+import { createHashFromFile } from "./utilities/createFileHash";
+import { logger } from "./utilities/logger";
```

Comment:
Introduced `logging` functionality with Winston to add some color to our logs

Might be a bit overkill but as long we we use it within future scripts as well I think its OK. I had a look at `chalk` as well but there are some compatibility issues.